src/Components/Game/Gameboard.tsx

```diff
@@ -23,11 +25,12 @@ const GameBoard = () => {
   const [players, setPlayers] = useState<Player[]>([])
   const [drawerIdx, setDrawerIdx] = useState(0)
   const [roomExists, setRoomExists] = useState(false)
-  const [phase, setPhase] = useState("")
+  const [phase, setPhase] = useState<Phase>("Pick_Word")
   const [loading, setLoading] = useState(true)
   const [round, setRound] = useState(1)
   const [wordChoices, setWordChoices] = useState<string[]>([])
   const [currentWord, setCurrentWord] = useState("")
+  const [pictureUrls, setPictureUrls] = useState<{sessionId: string, pictureUrl: string}[]>([])
```

Comment:
in the socket.on("send_game_info_to_client") you need to setSetpictureUrls with the current urls from the backend. This is for when a user reconnects so they have the picture urls